components/Withdraw.tsx

```diff
@@ -17,9 +18,24 @@ const Withdraw: React.FC<{}> = () => {
     text: "",
     hash: "",
   });
-  const { Contracts, Account }: any = useBlockchain();
+  const [estimatedFee, setEstimatedFee] = useState(0);
+  const { Contracts, Account, Signer }: any = useBlockchain();
   const { signer, selectedAccount, api, balances }: any = useWeb3();
 
+  //Estimate fee
+  useEffect(() => {
+    (async () => {
+      const gasEstimate = (await Signer.getGasPrice()).toString();
```

Comment:
_amount_ of gas used is around `128,895` based on testnet txs.
can update this number after doing a mainnet 
withdraw transactions.

all together should be smth like:
`verficationFee + gasPrice * 128895`